### PR TITLE
[stable/rabbitmq] Environment variables in rabbitmq container

### PR DESCRIPTION
**Pull request by PaulFarver**
#### What this PR does / why we need it:

- env variables: some rabbitmq configuration is done through [env variables](https://www.rabbitmq.com/configure.html#customise-environment). For example it might be necessary to control the CONFIG_FILE variable, when using an image that isn't based on the default bitnami image

#### Special notes for your reviewer:

This is very similar to #14932 for the rabbitmq-ha chart.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.1.6
+version: 6.1.7
```
Comment (tompizmor) [inline stable/rabbitmq/Chart.yaml]: According to semver versioning, this version should be bumped to `6.2.0`

**Review by tompizmor — CHANGES_REQUESTED** [review]
Other than the version bump, the PR looks good to me

File: stable/rabbitmq/Chart.yaml (hunk 1 above)
Comment (PaulFarver) [reply stable/rabbitmq/Chart.yaml]: Corrected in bd48a26fb1f4c1e208efc4b486b985b18c197156

**Pull request merged**